packages/trader/src/Modules/CFD/index.js

```diff
@@ -1,5 +0,0 @@
-import CFDDashboard from './Containers/cfd-dashboard';
```

Comment:
Can we remove the entire CFD folder in Modules?

Reply:
It is already done but for some reason github shows it differently.